mkdocs/technical/plugins/settings.md

```diff
@@ -107,4 +107,4 @@ where `settings` is a dictionary, a JSON schema or a Pydantic `BaseModel` descri
 
 If you need even more customization for your settings you can totally override how they are saved and loaded.
 Take a look at the `save_settings` and `load_settings` functions (always to be decorated with `@plugin`).  
-From there you can call external servers or devise a totally different format to store and load your settings. The Cat will call those functions and delegate to them how settings are managed instead of using a `settings.json` file.
\ No newline at end of file
+From there you can call external servers or devise a totally different format to store and load your settings. The Cat will call those functions and delegate to them how settings are managed instead of using a `settings.json` file.
```

Comment:
I have no idea what github is seeing being changed here... I did not touched this line. Maybe it's some white space issue(?)

Reply:
Indeed I didn't understand too, but no problem btw

Reply:
I was lazy and used the GitHub editor... and this was the result XD